get-started-with-tidb-lightning.md

```diff
@@ -41,13 +41,13 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
 
 ### 第 1 步：部署 TiDB 集群
 
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群（版本要求 2.0.9 以上），本教程使用 TiDB 4.0.3 版本。部署方法可参考 [TiDB 部署方式](https://docs.pingcap.com/zh/tidb/v3.0/overview#部署方式)。
+在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.0.0 版本为例，具体部署方法可参考[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
 
 ### 第 2 步：下载 TiDB Lightning 安装包
 
-通过以下链接获取 TiDB Lightning 安装包（选择与 TiDB 集群相同的版本）：
+通过以下链接获取 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）：
 
-- **v4.0.3**: [tidb-toolkit-v4.0.3-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v4.0.3-linux-amd64.tar.gz)
+- **v5.0.0-nightly**: [tidb-toolkit-v5.0.0-nightly-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.0.0-nightly-linux-amd64.tar.gz)
```

Comment:
@glorv can the nightly version be used for production?

Reply:
There is no `5.0.0` version available now. So maybe we need to merge this PR after `5.0.0-ga` version is released?

Reply:
OK

Reply:
```suggestion
- **v5.0.0**: [tidb-toolkit-v5.0.0-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.0.0-linux-amd64.tar.gz)
```
my suggestion after 5.0 GA ⬆️